### PR TITLE
Update 08-bsc ExportMetadata

### PR DESCRIPTION
**Pull request by gnehcein**

### CODE REVIEW

File: modules/tibc/light-clients/08-bsc/types/client_state.go [hunk 1]
```diff
@@ -80,9 +84,23 @@ func (m ClientState) Status(
...
 func (m ClientState) ExportMetadata(store sdk.KVStore) []exported.GenesisMetadata {
-	//TODO
-	return nil
+	gm := make([]exported.GenesisMetadata, 0)
+	callback := func(key, val []byte) bool {
+		gm = append(gm, clienttypes.NewGenesisMetadata(key, val))
+		return false
+	}
+
+	IteratorTraversal(store, host.KeyConsensusStatePrefix, callback)
```
Comment (chengwenxi) [inline modules/tibc/light-clients/08-bsc/types/client_state.go:96]: no need to export consensusState here, @dreamer-zq pls ensure it.

File: modules/tibc/light-clients/08-bsc/types/store.go [hunk 2]
```diff
@@ -18,6 +21,25 @@ var (
...
+
+	for ; iterator.Valid(); iterator.Next() {
+		if cb(iterator.Key(),iterator.Value()) {
+			break
+		}
+	}
+}
+
+
+
```
Comment (chengwenxi) [inline modules/tibc/light-clients/08-bsc/types/store.go:42]: remove

**Review by chengwenxi — COMMENTED** [review]
pls add some tests and run `make format` before committing.

File: modules/tibc/light-clients/08-bsc/types/client_state.go [hunk 3]
```diff
@@ -80,9 +93,22 @@ func (m ClientState) Status(
 	return exported.Active
 }
 
+// ExportMetadata exports all the processed times in the client store so they can be included in clients genesis
+// and imported by a ClientKeeper
```
Comment (chengwenxi) [inline modules/tibc/light-clients/08-bsc/types/client_state.go:97]: pls update this comment.

**Review by dreamer-zq — APPROVED** [review]

**Pull request merged**